### PR TITLE
[FIX] website_form: fix the website from issues

### PR DESCRIPTION
**Pull request by ppr-odoo**
Pad:https://pad.odoo.com/p/r.8ce0d17a6ed8033d55753c5bec4ec61f
Task:https://www.odoo.com/web#id=2082970&action=327&model=project.task&view_type=form&menu_id=4720

### CODE REVIEW

File: addons/website_crm/static/src/js/website_crm_editor.js [hunk 1]
```diff
@@ -14,8 +14,8 @@ FormEditorRegistry.add('create_lead', {
         type: 'many2one',
         relation: 'crm.team',
         domain: [['use_opportunities', '=', true]],
-        string: _t('Sales Channel'),
-        title: _t('Assign leads/opportunities to a sales channel.'),
+        string: _t('Sales Team'),
+        title: _t('Assign leads/opportunities to a sales Team.'),
```
Comment (mreficent) [inline addons/website_crm/static/src/js/website_crm_editor.js]: shouldn't `.pot` file be updated?

Comment (ppr-odoo) [reply addons/website_crm/static/src/js/website_crm_editor.js]: @mreficent done

File: addons/website_form/controllers/main.py [hunk 2]
```diff
@@ -36,6 +36,10 @@ def website_form(self, model_name, **kwargs):
             id_record = self.insert_record(request, model_record, data['record'], data['custom'], data.get('meta'))
             if id_record:
                 self.insert_attachment(model_record, id_record, data['attachments'])
+                # in case of an email, we want to send it immediately instead of waiting
+                # for the email queue to process
+                if model_name == 'mail.mail':
+                    request.env[model_name].sudo().browse(id_record).send()
```
Comment (fja-odoo) [inline addons/website_form/controllers/main.py]: insert_record could return the actual record instead of it's id now so no browse is needed

**Comment by fja-odoo** [comment]
@qsm-odoo looks good to me.

File: addons/website_form/controllers/main.py (hunk 2 above)
Comment (qsm-odoo) [reply addons/website_form/controllers/main.py]: @fja-odoo We're in stable here, so no ;)

Comment (ppr-odoo) [reply addons/website_form/controllers/main.py]: @qsm-odoo @fja-odoo  I've created a master branch https://github.com/odoo/odoo/pull/42390

**Pull request closed without merging**